### ISSUE TITLE
Customers not charged after trial period

### ISSUE THREAD

**Issue by APdevelopments** [osiset/laravel-shopify#482]
Hi, I have a few customers but it's not charged after 3 days trial period. How to handle that.

CONFIG:
`    'billing_enabled' => (bool) env('SHOPIFY_BILLING_ENABLED', 1),
`

Route: `Route::get('/', 'ShopifyController@catalogs')->middleware(['auth.shopify', 'billable'])->name('home');`

Store accepts payment and use the app in trial period but after the trial period store is not charged. Why?

**Comment by Kyon147** [comment]
Customers are only "charged" when their invoice billing period happens. So even if their trial period ends you won't see anything until Shopify invoices them.

For example, 

- Customer installs the app on a three day trial period.
- The trial period ends and goes onto regular billing.
- Shopify invoice the shop 20 days later (this depends on when the shop was set up)
- Your charge is added to their invoice as part of the total bill
- Shop pays invoice
- Shopify adds your cut of the money to your payouts section.

Basically, as long as you see "Reoccurring Charge Accepted" in your app details page, it means you get paid. You just need to be patient as payouts are all based on when the shop is charged by Shopify not when the trial period ends. 

Closing the issue as at the moment does not seem like a bug.
